lapis2/src/test/kotlin/org/genspectrum/lapis/controller/LapisControllerTest.kt

```diff
@@ -44,6 +57,7 @@ class LapisControllerTest(@Autowired val mockMvc: MockMvc) {
             .andExpect(jsonPath("\$[0].count").value(0))
             .andExpect(jsonPath("\$[0].country").value("Switzerland"))
             .andExpect(jsonPath("\$[0].age").value(42))
+            .andExpect(MockMvcResultMatchers.header().stringValues("Lapis-Data-Version", "1234"))
```

Comment:
Please add a static import, so that it reads:
```suggestion
            .andExpect(header().stringValues("Lapis-Data-Version", "1234"))
```